dali/operators/expressions/arithmetic.h

```diff
@@ -229,6 +236,9 @@ class ArithmeticGenericOp : public Operator<Backend> {
     if (!types_layout_inferred_) {
       result_type_id_ = PropagateTypes<Backend>(*expr_, ws);
       result_layout_ = GetCommonLayout<Backend>(*expr_, ws);
+      std::vector<ExprConstant *> constant_nodes;
+      GetConstantNodes(*expr_, constant_nodes);
+      constant_storage_.Initialize(spec_, ws.has_stream() ? ws.stream() : 0, constant_nodes);
```

Comment:
I don't know if we need to:
```
ws.has_stream() ? ws.stream() : 0
```
Or just relay that is `!has_stream()` then `ws.stream()` is 0.

Reply:
It's not 0 for CPU Backend. It throws you an error if you don't have a stream assigned.